CHANGELOG

```diff
@@ -15,6 +15,7 @@ v3.17 (Month 2020)
   * Improve current implementation of persisting data when creating and editing Card, Evidence, Issue, and Note
     - Data now persists for the whole form instead of only the textarea
     - Data will be cleared when the "Cancel" link is clicked
+    - Extend functionality to work with templates
```

Comment:
This is a developer note, in this PR you're extending what you added in the base branch, but in the base branch you created this. From a user's point of view, they're getting the whole functionality at the same time, we're not "Extending" anything, we're adding from scratch a functionality or modifying it.

How can we describe what this PR does in a way the user cares about / understands it?